frontend/src/components/pages/volunteer/shift/VolunteerShiftsPage.tsx

```diff
@@ -58,7 +58,7 @@ const VolunteerShiftsPage = (): React.ReactElement => {
         defaultIndex={VolunteerPages.VolunteerShifts}
         tabs={VolunteerNavbarTabs}
       />
-      <Box bg="background.light" p={10} h="100vp">
+      <Box bg="background.light" p={10}>
```

Comment:
```suggestion
      <Box bg="background.light" p={10} minH="100vp">
```
maybe something like this might be more appropriate?

Reply:
Also not sure if 100vp is supposed to be 100vh. Investigating.

Reply:
nvm, we don't need this all.